_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -50424,6 +50502,26 @@
 	},
 /turf/simulated/floor/plating,
 /area/station/security/armory/secure)
+"jnI" = (
+/obj/effect/turf_decal/stripes/asteroid/end{
+	dir = 1
+	},
+/obj/structure/cable/orange{
+	d2 = 8;
+	icon_state = "0-8"
+	},
+/obj/machinery/power/apc{
```

Comment:
Use the `/obj/machinery/power/apc/important/directional` subtypes for the cryo and arrivals APCs.

Reply:
fixed